### PR TITLE
React routes : Multi route SPA

### PR DESCRIPTION
**Pull request by devendra-alt**
### 👨‍💻  Requirments fulfilled
✔️ Created two components `Header`, and `Home`
✔️ Used react-router package for routing 
✔️ Added react-router paths for various components 
✔️ Displayed navigation header on every components head
✔️ Added abortController for quote fetch to avoid unnecessary calls

### CODE REVIEW

File: src/components/calculator.js
Comment (AbelG101) [inline src/components/calculator.js:1]: - [ ] Great effort! However, as per the project requirements, it's important to ensure that the layout design matches the provided wireframe. Specifically, for the "Calculator" page, the wireframe specifies a 2-column layout with the text on the left and the calculator on the right. To meet this requirement, please adjust the layout accordingly. Thank you for your attention to detail!
- Wireframe:
  
- Your design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @devendra-alt 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Proper implementation of React router ✔
- Navigation-link routes to the proper page ✔
- Correct use of Git Flow ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js
Comment (devendra-alt) [reply src/components/calculator.js:1]: Thank you @AbelG101 💯 , for valuable feedback

**Review by muneebulrehman — APPROVED** [review]
Hi @devendra-alt , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Good follow-up with the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**